pi-docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
   database:
     image: "postgres" # use latest official postgres version
     env_file:
-      - database.env # configure postgres
+      - ./database.env # configure postgres
```

Comment:
Does the Pi need a git secret key? Or can someone manually move over a sql backup and decrypt it.

Reply:
The pi does not need a git secret key. I'll dm you about this